contributors/devel/conformance-tests.md

```diff
@@ -0,0 +1,159 @@
+# Conformance Testing in Kubernetes
+
+The Kubernetes conformance test suite is a set of testcases, currently a
+subset of the integration/e2e tests, that the Architecture SIG has approved
+to define the core set of interoperable features that all Kubernetes
+deployments must support.
+
+Contributors must write and submit e2e tests first (approved by owning Sigs). 
+Once the new tests prove to be stable in CI runs, later create a follow up PR 
+to add the test to conformance. This approach also decouples the development 
+of useful tests from their promotion to conformance.
+
+A conformance test verifies the expected functionality works as a user might encounter it in the wild, 
+and tests should begin by covering the most important and visible aspects of the function.
+
+Conformance tests in the default profile should cover features that are:
```

Comment:
Maybe drop this bulleted list under a heading like "Conformance Test Requirements"

I want to be able to say something like "a test is eligible for promotion to conformance if it meets these requirements [link to bulleted list]"

Reply:
fixed